<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrikazCreate</class>
 <widget class="QWidget" name="PrikazCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>86</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>86</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Создание приказа</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="prikaz_create_cancel">
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="prikaz_create_create">
     <property name="text">
      <string>Создать приказ</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDateEdit" name="prikaz_create_date"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Выберите дату создания приказа</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
